hw/ip/otbn/dv/model/iss_wrapper.cc

```diff
@@ -322,6 +334,17 @@ void ISSWrapper::start(uint32_t addr) {
   run_command(oss.str(), nullptr);
 }
 
+void ISSWrapper::edn_rnd_data(uint32_t edn_rnd_data[8]) {
+  std::ostringstream oss;
+  oss << "edn_rnd_data " << wlen_val_to_hex_str(edn_rnd_data);
+  oss << "\n";
```

Comment:
Nit: Probably better to chain this into one statement (`oss << "edn_rnd_data " << wlen_val_to_hex_str(edn_rnd_data) << "\n"`)